src/js/containers/bulkDownload/archive/AwardDataArchiveContainer.jsx

```diff
@@ -0,0 +1,189 @@
+/**
+ * AwardDataArchiveContainer.jsx
+ * Created by Lizzie Salita 12/14/17
+ */
+
+import React from 'react';
+import moment from 'moment';
+import * as BulkDownloadHelper from 'helpers/bulkDownloadHelper';
+
+import AwardDataArchiveContent from 'components/bulkDownload/archive/AwardDataArchiveContent';
+
+const columns = [
+    {
+        columnName: 'agency',
+        displayName: 'Agency'
+    },
+    {
+        columnName: 'url',
+        displayName: 'Archive File'
+    },
+    {
+        columnName: 'fy',
+        displayName: 'Fiscal Year'
+    },
+    {
+        columnName: 'date',
+        displayName: 'Date As Of'
+    }
+];
+
+export default class AwardDataArchiveContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            inFlight: true,
+            agencies: {
+                cfoAgencies: [],
+                otherAgencies: []
+            },
+            filters: {
+                agency: {
+                    id: 'all',
+                    name: 'All'
+                },
+                type: {
+                    name: 'contracts',
+                    display: 'Contracts'
+                },
+                fy: '2018'
```

Comment:
Can we utilize our FY helper to either generate a current fiscal year by default or put a hardcoded value in the fiscalYearHelper with the other hardcoded years?